test-configs.yaml

```
@@ -665,6 +665,11 @@ device_types:
     dtb: 'freescale/imx8mp-evk.dtb'
     boot_method: uboot
 
+  imx8mq-zii-ultra-zest:
+    mach: imx
+    class: arm64-dtb
```

The `imx` boards in `arm64` are all under `freescale` and don't follow the convention defined by the `arm64-dtb` class, which uses the mach name (`imx` in this case).  This fixes the prolem:
```suggestion
    arch: arm64
    dtb: 'freescale/imx8mq-zii-ultra-zest.dtb'
```

---

With the change above, I could get a staging job running fine:
https://hekla.openlab.pengutronix.de/scheduler/job/387286
https://staging.kernelci.org/test/plan/id/5fa07d66723bf044100145a8/